docs_src/data_block.ipynb

```diff
@@ -1023,6 +1023,356 @@
     "  - [`CollabList`](/collab.html#CollabList) for collaborative filtering"
```

Comment:
<p>I'd show that there are new items inside too.</p>

 _Reply via <a href='https://app.reviewnb.com/fastai/fastai/pull/2026/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='78'/>

Reply:
I don't get this comment, could you elaborate on the changes I need to do? thanks!

Reply:
Ok, I got this, thanks!